src/helperfuncs.cpp

```diff
@@ -432,6 +432,9 @@ bool InspIRCd::IsValidDuration(const std::string& duration)
 
 std::string InspIRCd::DurationString(time_t duration)
 {
+	if (duration == 0)
+		return "0";
```

Comment:
```suggestion
		return "0s";
```

Reply:
I initially did that, but that looks weird in the current use case. "+H 2:0s" doesn't look as proper as "+H 2:0". "0s" may fit better if/when other functions hit this same scenario though.

Reply:
I feel that always  showing the seconds with a duration specifier is more user friendly as it allows users who aren't familiar with what the mode does to infer what the parameter is for.